Packs/Mimecast/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __MimecastV2__
+Fixed an issue where added users by emails showed an error though performed successfully.
```

Comment:
Fixed an issue where added users by emails returned an error although executed successfully.

Is “added users by emails” a command? If not try: “users added via emails”

Reply:
you can add a user via domain or via email. changing.